satellite/satellitedb/overlaycache.go

```diff
@@ -146,6 +146,33 @@ func (cache *overlaycache) SelectNewStorageNodes(ctx context.Context, count int,
 	return nodes, nil
 }
 
+// GetNodeIPs returns a list of node IP addresses.
+func (cache *overlaycache) GetNodeIPs(ctx context.Context, nodeIDs []storj.NodeID) (nodeIPs []string, err error) {
+	defer mon.Task()(&ctx)(&err)
+
+	var rows *sql.Rows
+	// warning: these node IP addresses might be returned out of order
```

Comment:
nit - should this warning be part of the documentation comment above the function?